linode/manager
# refactor: [M3-8766] - Cleanup feature flag - Create Using Command Line (DX Tools Additions)

## Description 📝
This PR removes feature flag used for the Create Using Command Line (DX Tools Additions) and cleanup its usages. 

## Changes  🔄
List any change relevant to the reviewer.
- ...
- ...

## Target release date 🗓️
10/28


## How to test 🧪


### Verification steps
(How to verify changes)
- Navigate to the linode create flow
- Fill all required fileds and click on view code snippets or create using command line button
- Verify there is no regression with prod. 
- Verify feature flag is removed. (Flag from LD will be deprecated after 10/28)

## As an Author I have considered 🤔

*Check all that apply*
